visidata/statusbar.py

```diff
@@ -153,7 +153,7 @@ def drawLeftStatus(vd, scr, vs):
 @VisiData.api
 def rightStatus(vd, sheet):
     'Return right side of status bar.  Overrideable.'
-    return options.disp_rstatus_fmt.format(sheet=sheet, vd=vd)
+    return MissingAttrFormatter().format(options.disp_rstatus_fmt, sheet=sheet, vd=vd)
```

Comment:
Also let's use `sheet.options.disp_rstatus_fmt`, which will allow different sheet classes to have different options.